Framework/Core/include/Framework/DataTypes.h

```diff
@@ -134,6 +134,30 @@ constexpr std::array<float, 5> trackQAScaleGloP1{0.183731, 0.409071, 0.00621802,
 constexpr std::array<float, 2> trackQAScaledTOF{1.1, 0.33};
 } // namespace o2::aod::track
 
+namespace o2::aod::mctracklabel
+{
+// ! Bit mask to indicate detector mismatches (bit ON means mismatch). Bit 0-6: mismatch at ITS layer. Bit 7-9: # of TPC mismatches in the ranges 0, 1, 2-3, 4-7, 8-15, 16-31, 32-63, >64. Bit 10: TRD, bit 11: TOF, bit 15: indicates negative label
+enum McMaskEnum : uint16_t {
```

Comment:
What is the correspondence between these enums and what is actually filled? Note that the ITS track per see may have no fakes (all clusters having the same label) but can be matched to TPC track with a different label. In this case ITS-TPC label will have `isFake()==true`. With your PR it will be flagged in bit0, why?